commit 38cd40d389fbe65c5d83f6102fe27ddbd2cb1cff (refs/changes/56/4856/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-09-20T19:56:24+02:00 (1 year, 1 month ago)
    
    TRITON-784 Add requirements.bootrom to the list of allowed image fields in CloudAPI
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/lib/datasets.js b/lib/datasets.js
index 05106a2..22f311f 100644
--- a/lib/datasets.js
+++ b/lib/datasets.js
@@ -147,7 +147,8 @@ function translate(req, dataset) {
             ['max_ram', 'max_ram'],
             ['min_memory', 'min_ram'],
             ['min_ram', 'min_ram'],
-            ['brand', 'brand']
+            ['brand', 'brand'],
+            ['bootrom', 'bootrom']
         ].forEach(function (mapping) {
             if (dsetReqs[mapping[1]]) {
                 objReqs[mapping[0]] = dsetReqs[mapping[1]];
